components/playground/instance/tiflash_pre7.go

```diff
@@ -146,7 +146,7 @@ func (inst *TiFlashInstance) checkConfigOld(deployDir, clusterManagerPath string
 	}()
 
 	// Write default config to buffer
-	if err := writeTiFlashConfigOld(flashBuf, version, inst.TCPPort, inst.Port, inst.ServicePort, inst.StatusPort,
+	if err := writeTiFlashConfigOld(flashBuf, version, inst.Port, inst.ServicePort, inst.StatusPort,
```

Comment:
I think we can not remove tcp port before tiflash 7.0

Reply:
Right, it'll cause single-node multi-instance conflict on 9000 port. Thanks for reminding.

Addressed.